app/styleguide/page.tsx

```diff
@@ -538,14 +1015,14 @@ function Page() {
 
                 <div><pre><code>
                   {`import {
```

Comment:
Same comment as above. The lack of indentation was intentional. Does it still render correctly on the page?